balancer/frontend-v2
# Chore: Move initial swap tokens to config

# Description

- Move the initial swap tokens for each network from the initialTokens.json file to the individual config files. 

## Type of change


## How should this be tested?

- Load each network in a private browser (so you don't have localstorage overwriting the chosen tokens)
- Navigate to swap page and ensure the initial tokens are the ones set for that network. 

## Checklist:
